docs/general/ad-filtering/create-own-filters.md

```diff
@@ -445,9 +445,15 @@ The modifier `":" h_value` part may be omitted. In that case the modifier matche
 * `@@||example.com^$header=set-cookie:/foo\, bar\$/` unblocks requests whose responses have the `Set-Cookie` header with value matching the `foo, bar$` regular expression.
 * `@@||example.com^$header=set-cookie` unblocks requests whose responses have a `Set-Cookie` header with any value.
 
+:::caution Limitations
+
+In Adguard Browser Extension, [`$header`](#header-modifier) modifier is only compatible with [`$csp`](#csp-modifier) and [`$removeheader`](#removeheader-modifier) advanced modifiers. Rules that use [`$header`](#header-modifier) modifier with one of these, are subject to allowlisting logic of advanced modifiers.
```

Comment:
We should explain the difference in matching logic of this modifier. It is matched only when headers are received so if the request was blocked or redirected on an earlier stage nothing will be applied. This logic is similar for both the browser extension and CoreLibs.